ðŸ”„ Starting training with config: {'activation': 'sigmoid', 'batch_size': 64, 'dataset': 'fashion_mnist', 'epochs': 5, 'hidden_size': 64, 'learning_rate': 0.0001, 'loss': 'mean_squared_error', 'num_layers': 5, 'optimizer': 'sgd', 'weight_decay': 0.5, 'weight_init': 'Xavier'}
Epoch 1/5 - loss: 0.4658 - accuracy: 0.1002 - val_loss: 0.4650 - val_accuracy: 0.0983
Epoch 2/5 - loss: 0.4639 - accuracy: 0.1002 - val_loss: 0.4632 - val_accuracy: 0.0983
Epoch 3/5 - loss: 0.4622 - accuracy: 0.1002 - val_loss: 0.4617 - val_accuracy: 0.0983
Epoch 4/5 - loss: 0.4608 - accuracy: 0.1002 - val_loss: 0.4604 - val_accuracy: 0.0983
Epoch 5/5 - loss: 0.4596 - accuracy: 0.1002 - val_loss: 0.4592 - val_accuracy: 0.0983
Test loss: 0.4590 - Test accuracy: 0.1000
K:\DA6401\da6401_assignment1\train.py:118: UserWarning: Starting a Matplotlib GUI outside of the main thread will likely fail.
  plt.figure(figsize=(10, 8))
K:\DA6401\da6401_assignment1\train.py:270: UserWarning: Starting a Matplotlib GUI outside of the main thread will likely fail.
  fig, axs = plt.subplots(2, 5, figsize=(12, 6))
